.github/renovate.json

```
@@ -0,0 +1,8 @@
+{
+  "automerge": true,
+  "enabledManagers": ["github-actions", "npm"],
+  "labels": ["dependencies", "javascript"],
+  "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
+  "stabilityDays": 3,
```

Reading through the [Renovate docs on `stabilityDays`](https://docs.renovatebot.com/configuration-options/#stabilitydays) and it looks like [using with `internalChecksFilter`](https://docs.renovatebot.com/configuration-options/#suppress-branchpr-creation-for-x-days) is the safest approach. We wouldn't want a malicious package to run in our CI if we can't help it. Deduping packages that release a new fixed version quickly (in <3 days) after a previous release is nice too.

```suggestion
  "internalChecksFilter": "strict",
  "stabilityDays": 3,
```

Thoughts?

https://docs.renovatebot.com/configuration-options/#internalchecksfilter

---

Yes, it is also advised by the developers, so it is preferable to utilise it.👍